pkgs/development/tools/grabserial/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchgit, buildPythonPackage, pythonPackages}:
+
+buildPythonPackage rec {
+
+ name = "grabserial-8b9c98ea";
```

Comment:
A git commit id is not a good version number (there is no meaningful way to compare two hashes). If a package has no release number, we tend to use the date of the commit instead (YYYYMMDD).